src/components/BorderlessInput.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react'
+import './BorderlessInput.css'
+
+interface Props {
+    id: string
+    placeholder: string
+    value: string
+    onChange: (value: string) => void
+}
+
+interface State {
+    active: boolean
+}
+
+class component extends React.Component<Props> {
+    state: State = {
+        active: false
+    }
+
+    onFocusInput = () => {
+        console.log(`onClickInput`)
+        this.setState({
+            active: true
+        })
+    }
+
+    onBlurInput = () => {
+        console.log(`onBlur`)
```

Comment:
left this in on purpose?

Reply:
Nope, will remove, pushed this early to get feedback on the change detection or save type issue an should have cleaned it up first.